src/__tests__/authenticate.test.ts

```diff
@@ -0,0 +1,23 @@
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import * as apiClient from '../';
+import { validData, invalidData, token as tokenAuth } from './fixtures/authenticate';
+
+const mock = new MockAdapter(axios);
+
+afterEach(() => {
+	mock.reset();
+});
+
+const apiClientInit = new apiClient.APIClient({ apiBase: 'unics-social' });
+
+test('authenticate(): authenticates valid users correctly', async () => {
+	mock.onPost('unics-social/authenticate').reply(200, { token: tokenAuth });
```

Comment:
Currently, this will mean posting any data to the authenticate endpoint will give a valid token. Can you make the mock use specific body data? If you visit [https://github.com/ctimmerm/axios-mock-adapter](https://github.com/ctimmerm/axios-mock-adapter) and CTRL+F for "Mocking a request with a specific request body/data" you should find it there.

Can you also update the test below too? This makes sure that both tests are actually testing the client is passing the right info to the API - right now they just test that the client processes a response correctly.